cpp/src/arrow/util/compression_brotli.cc

```diff
@@ -31,10 +31,6 @@
 namespace arrow {
 namespace util {
 
-// Brotli compression quality is max (11) by default, which is slow.
-// We use 8 as a default as it is the best trade-off for Parquet workload.
-constexpr int kBrotliDefaultCompressionLevel = 8;
```

Comment:
IMHO it was fine to leave this default here.

Reply:
I would like to not do that and keep all of the levels in one place.
Also, some codec source files are not always compiled into the project and thus there would have been an extra ifdef necessary just because of that in the function for retrieving the default compression level for the given compression type.